src/main/java/io/openliberty/tools/intellij/util/LibertyProjectUtil.java

```diff
@@ -149,6 +153,27 @@ public static ShellTerminalWidget getTerminalWidget(Project project, LibertyModu
         TerminalView terminalView = TerminalView.getInstance(project);
         // look for existing terminal tab
         ShellTerminalWidget widget = getTerminalWidget(libertyModule, terminalView);
+        TerminalToolWindowManager manager = TerminalToolWindowManager.getInstance(project);
+        ToolWindow toolWindow = manager.getToolWindow();
+
+        if (toolWindow != null) {
+            ContentManager contentManager = toolWindow.getContentManager();
+            Content[] contents = contentManager.getContents();
+
+            int index = 0;
+            for (int i = 0; i < contents.length; i++) {
+                if (contents[i].getPreferredFocusableComponent().equals(widget)) {
```

Comment:
This may no longer apply once the code has been refactored, but consider if `widget == null`. There should be guard for this entire block so that we don't spend time searching for a component that won't be found.

Reply:
@mrglavas , Now Its checking `widget == null` in the if condition

Reply:
Thanks. I see that.